pkg/util/util.go

```diff
@@ -1544,29 +1544,33 @@ func GetEnvWithDefault(key string, defaultval string) string {
 	return val
 }
 
+func IsInvalidKubeConfigError(err error) bool {
```

Comment:
This function name is confusing in terms of what it is returning. Maybe it should just be `IsInvalidKubeConfig` or if you are going with the other suggestion about moving err check and returning err part(see below) here, then keep it as is.
```go
		if util.IsInvalidKubeConfigError(err) {
			return fmt.Errorf("invalid KUBECONFIG provided. Please point to a valid KUBECONFIG. You do not have to be logged in %w", err)
		}
```